calico-enterprise/reference/installation/tigerastatus.mdx

```diff
@@ -0,0 +1,42 @@
+---
+description: Descriptions of Tigera Status fields
+---
+
+# Tigera status
+
+Installing {{prodname}} on your Kubernetes cluster is managed by the Tigera operator. The Tigera operator is deployed as a ReplicaSet in the `tigera-operator` namespace, and records status in a custom resource named, `tigerastatus`. The operator get its configuration from several Custom Resources (CRs); the central one is the Installation CR.
+
+To check tigerastatus, use the following command.
+
+`kubectl get tigerastatus`
+
+## Log storage controllers
+
+The following controllers comprise the log storage component. Note the following:
+
+- All controllers are required and are not configurable
+- Controllers work together to implement log storage functionality, but are reconciled independently
+- Controllers do not communicate directly with each other; they communicate through the Kubernetes API
+
+To check log collector status, run the following command:
+
+`kubectl get logcollector.operator.tigera.io tigera-secure`
+
+The following table lists controller descriptions.
+
+| Tigera status field         | Description                                                  |
+| --------------------------- | ------------------------------------------------------------ |
+| log-storage                 | Provides initialization validation and defaulting for log storage, creates base namespaces for controllers to deploy to, and provides status and condition updates on other controllers. |
+| log-storage-elastic         | Installs the ECK operator and Elasticsearch CR into the cluster (and Kibana if configured). Runs only on standalone clusters and the management cluster (if you are using the multi-cluster management feature). |
+| log-storage-kubecontrollers | Installs Elasticsearch kube-controllers.                     |
+| log-storage access          | Installs the {{prodname}} generalized log storage API service. |
+| log-storage-secrets         | Provisions all required key pairs and trusted bundles for log storage components to operate. |
+| log-storage-users           | Generates unique credentials for each log storage instance and provisions users for Elasticsearch. |
+
+
```

Comment:
Do we want to talk about the other non-log-storage TigeraStatus resources that the operator creates?

From a quick glance at the code, some or all of these may show up depending on how the user configures their cluster:

- calico
- apiserver
- manager
- intrusion-detection
- compliance
- log-collector
- applicationlayer
- management-cluster-connection
- policy-recommendation
- egressgateway
- calico-windows
- authentication
- secrets

Reply:
Yes, I was thinking the same thing...log storage is first one in, but allows us to expand. I'll redo intro to accommodate.

Reply:
I think we should. Even though we'll only provide status for log-storage for now. If Support and users find this valuable, they can ask for full documentation.